cciskel-duffy

```diff
@@ -21,7 +22,7 @@ import argparse
 
 url_base="http://admin.ci.centos.org:8080"
 api_key=open(os.path.expanduser('~/duffy.key')).read().strip()
-cached_state_path = os.path.expanduser('~/duffy-state-cache.json')
+cached_state_path = os.path.expanduser('~/.duffy-state-cache.json')
```

Comment:
We could make it configurable via cmdline/environment too.  But I think the end state is that we store this state in Duffy itself, right?  So can we keep the current location to avoid disruption for me until we have the real fix there?